K번째수.js

```diff
@@ -0,0 +1,16 @@
+function solution(array, commands) {
+    let answer = [];
+    for (let count1 = 0; count1 < commands.length; count1++) {
+        const i = commands[count1][0] - 1;
+        const j = commands[count1][1] - 1;
+        const k = commands[count1][2] - 1;
+
+        let newArray = [];
+        for (let count2 = i; count2 < j + 1; count2++) {
+            newArray.push(array[count2]);
+        }
+        newArray.sort((a, b) => a - b);
+        answer.push(newArray[k]);
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(array, commands) {
    const answer = [];
    
    for (let i = 0; i < commands.length; i++) {
        const startIdx = commands[i][0] - 1;
        const lastIdx = commands[i][1] - 1;
        const selectIdx = commands[i][2] - 1;
        const newArray = [];
        
        for (let j = startIdx; j < lastIdx + 1; j++) {
            newArray.push(array[j]);
        }
        newArray.sort((a, b) => a - b);
        answer.push(newArray[selectIdx]);
    }
    
    return answer;
}
```
결이 다른 코드는 한줄 띄워주시고 배열이나 오브젝트같은 참조타입은 const로 선언해주셔야 합니다. 그리고 for문 내의 변수명을 문제 설명에 i, j, k가 있어서 그러신거같은데 일반 for문의 변수명을 i, j, k순으로 바꾸고 문제에서 지정해준 i, j, k를 의미있는 변수명을 사용하는게 좋을거같아요.